conformance/tests/directives_deprecated_pep702_example.py

```diff
@@ -0,0 +1,44 @@
+"""
```

Comment:
I'm not sure it makes sense for this feature to have three separate test files in the conformance suite. It's a pretty straightforward feature, so it probably makes sense to combine them into a single test "directives_deprecated.py" that appears in the conformance summary as a single row. Just trying to keep the test granularity consistent between features.

In any case, the test name should not include the PEP number. The PEP is the original spec, but it is superseded by the typing spec. The fact that the feature was originally part of a PEP is a historical footnote only.